### DISCUSSION TITLE
SAGE206612

### DISCUSSION THREAD

**Discussion by Mucka911** [Koenkk/zigbee2mqtt#15514] [Devices]
Good day, 

First off greatly appreciate all the work being done with this project :) 

I am having a very hard time getting my SAGE doorbell to expose anything to HA. Battery, Action, and link quality are all null despite multiple attempts to reset and re connect. Any advice with this? 

LOG: https://pastebin.com/xvqpJv5p

`Info 2022-12-12 11:54:09Accepting joining not in blocklist device '0x0008890001d9af36'
Info 2022-12-12 11:54:09Device '0x0008890001d9af36' joined
Info 2022-12-12 11:54:09MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x0008890001d9af36","ieee_address":"0x0008890001d9af36"},"type":"device_joined"}'
Info 2022-12-12 11:54:09MQTT publish: topic 'homeassistant/sensor/0x0008890001d9af36/battery/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state","value_template":"{{ value_json.state }}"}],"device":{"identifiers":["zigbee2mqtt_0x0008890001d9af36"],"manufacturer":"EchoStar","model":"SAGE by Hughes doorbell sensor (SAGE206612)","name":"0x0008890001d9af36","sw_version":" 3.2.2"},"device_class":"battery","enabled_by_default":true,"entity_category":"diagnostic","name":"0x0008890001d9af36 battery","state_class":"measurement","state_topic":"zigbee2mqtt/0x0008890001d9af36","unique_id":"0x0008890001d9af36_battery_zigbee2mqtt","unit_of_measurement":"%","value_template":"{{ value_json.battery }}"}'
Info 2022-12-12 11:54:09MQTT publish: topic 'homeassistant/sensor/0x0008890001d9af36/action/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state","value_template":"{{ value_json.state }}"}],"device":{"identifiers":["zigbee2mqtt_0x0008890001d9af36"],"manufacturer":"EchoStar","model":"SAGE by Hughes doorbell sensor (SAGE206612)","name":"0x0008890001d9af36","sw_version":" 3.2.2"},"enabled_by_default":true,"icon":"mdi:gesture-double-tap","name":"0x0008890001d9af36 action","state_topic":"zigbee2mqtt/0x0008890001d9af36","unique_id":"0x0008890001d9af36_action_zigbee2mqtt","value_template":"{{ value_json.action }}"}'
Info 2022-12-12 11:54:09MQTT publish: topic 'homeassistant/sensor/0x0008890001d9af36/linkquality/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state","value_template":"{{ value_json.state }}"}],"device":{"identifiers":["zigbee2mqtt_0x0008890001d9af36"],"manufacturer":"EchoStar","model":"SAGE by Hughes doorbell sensor (SAGE206612)","name":"0x0008890001d9af36","sw_version":" 3.2.2"},"enabled_by_default":false,"entity_category":"diagnostic","icon":"mdi:signal","name":"0x0008890001d9af36 linkquality","state_class":"measurement","state_topic":"zigbee2mqtt/0x0008890001d9af36","unique_id":"0x0008890001d9af36_linkquality_zigbee2mqtt","unit_of_measurement":"lqi","value_template":"{{ value_json.linkquality }}"}'
Info 2022-12-12 11:54:09Starting interview of '0x0008890001d9af36'
Info 2022-12-12 11:54:09MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x0008890001d9af36","ieee_address":"0x0008890001d9af36","status":"started"},"type":"device_interview"}'
Info 2022-12-12 11:54:09Accepting joining not in blocklist device '0x0008890001d9af36'
Debug 2022-12-12 11:54:09Device '0x0008890001d9af36' announced itself
Info 2022-12-12 11:54:09MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x0008890001d9af36","ieee_address":"0x0008890001d9af36"},"type":"device_announce"}'
Debug 2022-12-12 11:54:09Received MQTT message on 'homeassistant/sensor/0x0008890001d9af36/battery/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state","value_template":"{{ value_json.state }}"}],"device":{"identifiers":["zigbee2mqtt_0x0008890001d9af36"],"manufacturer":"EchoStar","model":"SAGE by Hughes doorbell sensor (SAGE206612)","name":"0x0008890001d9af36","sw_version":" 3.2.2"},"device_class":"battery","enabled_by_default":true,"entity_category":"diagnostic","name":"0x0008890001d9af36 battery","state_class":"measurement","state_topic":"zigbee2mqtt/0x0008890001d9af36","unique_id":"0x0008890001d9af36_battery_zigbee2mqtt","unit_of_measurement":"%","value_template":"{{ value_json.battery }}"}'
Debug 2022-12-12 11:54:09Received MQTT message on 'homeassistant/sensor/0x0008890001d9af36/action/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state","value_template":"{{ value_json.state }}"}],"device":{"identifiers":["zigbee2mqtt_0x0008890001d9af36"],"manufacturer":"EchoStar","model":"SAGE by Hughes doorbell sensor (SAGE206612)","name":"0x0008890001d9af36","sw_version":" 3.2.2"},"enabled_by_default":true,"icon":"mdi:gesture-double-tap","name":"0x0008890001d9af36 action","state_topic":"zigbee2mqtt/0x0008890001d9af36","unique_id":"0x0008890001d9af36_action_zigbee2mqtt","value_template":"{{ value_json.action }}"}'
Debug 2022-12-12 11:54:09Received MQTT message on 'homeassistant/sensor/0x0008890001d9af36/linkquality/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state","value_template":"{{ value_json.state }}"}],"device":{"identifiers":["zigbee2mqtt_0x0008890001d9af36"],"manufacturer":"EchoStar","model":"SAGE by Hughes doorbell sensor (SAGE206612)","name":"0x0008890001d9af36","sw_version":" 3.2.2"},"enabled_by_default":false,"entity_category":"diagnostic","icon":"mdi:signal","name":"0x0008890001d9af36 linkquality","state_class":"measurement","state_topic":"zigbee2mqtt/0x0008890001d9af36","unique_id":"0x0008890001d9af36_linkquality_zigbee2mqtt","unit_of_measurement":"lqi","value_template":"{{ value_json.linkquality }}"}'
Info 2022-12-12 11:54:11Successfully interviewed '0x0008890001d9af36', device has successfully been paired
Info 2022-12-12 11:54:11Device '0x0008890001d9af36' is supported, identified as: EchoStar SAGE by Hughes doorbell sensor (SAGE206612)
Info 2022-12-12 11:54:11MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":{"description":"SAGE by Hughes doorbell sensor","exposes":[{"access":1,"description":"Remaining battery in %","name":"battery","property":"battery","type":"numeric","unit":"%","value_max":100,"value_min":0},{"access":1,"description":"Triggered action (e.g. a button click)","name":"action","property":"action","type":"enum","values":["bell1","bell2"]},{"access":1,"description":"Link quality (signal strength)","name":"linkquality","property":"linkquality","type":"numeric","unit":"lqi","value_max":255,"value_min":0}],"model":"SAGE206612","options":[],"supports_ota":false,"vendor":"EchoStar"},"friendly_name":"0x0008890001d9af36","ieee_address":"0x0008890001d9af36","status":"successful","supported":true},"type":"device_interview"}'
Debug 2022-12-12 11:54:11Retrieving state of '0x0008890001d9af36' after reconnect
Debug 2022-12-12 11:54:29Received Zigbee message from '0x0008890001d9af36', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":0,"fileVersion":9,"imageType":2,"manufacturerCode":4116}' from endpoint 18 with groupID 0
Debug 2022-12-12 11:54:29Device '0x0008890001d9af36' requested OTA
Debug 2022-12-12 11:54:35Received Zigbee message from '0x0008890001d9af36', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":0,"fileVersion":9,"imageType":2,"manufacturerCode":4116}' from endpoint 18 with groupID 0
Debug 2022-12-12 11:54:35Device '0x0008890001d9af36' requested OTA
Debug 2022-12-12 11:54:44Received Zigbee message from '0x0008890001d9af36', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":0,"fileVersion":9,"imageType":2,"manufacturerCode":4116}' from endpoint 18 with groupID 0
Debug 2022-12-12 11:54:44Device '0x0008890001d9af36' requested OTA
Debug 2022-12-12 11:54:56Received Zigbee message from '0x0008890001d9af36', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":0,"fileVersion":9,"imageType":2,"manufacturerCode":4116}' from endpoint 18 with groupID 0
Debug 2022-12-12 11:54:56Device '0x0008890001d9af36' requested OTA
Debug 2022-12-12 11:56:17Received Zigbee message from '0x0008890001d9af36', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":0,"fileVersion":9,"imageType":2,"manufacturerCode":4116}' from endpoint 18 with groupID 0
Debug 2022-12-12 11:56:17Device '0x0008890001d9af36' requested OTA
Debug 2022-12-12 11:56:23Received Zigbee message from '0x0008890001d9af36', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":0,"fileVersion":9,"imageType":2,"manufacturerCode":4116}' from endpoint 18 with groupID 0
Debug 2022-12-12 11:56:23Device '0x0008890001d9af36' requested OTA
Debug 2022-12-12 11:56:32Received Zigbee message from '0x0008890001d9af36', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":0,"fileVersion":9,"imageType":2,"manufacturerCode":4116}' from endpoint 18 with groupID 0
Debug 2022-12-12 11:56:32Device '0x0008890001d9af36' requested OTA
Debug 2022-12-12 11:56:44Received Zigbee message from '0x0008890001d9af36', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":0,"fileVersion":9,"imageType":2,"manufacturerCode":4116}' from endpoint 18 with groupID 0
Debug 2022-12-12 11:56:44Device '0x0008890001d9af36' requested OTA`

**Comment by k8gg** [comment]
`action` and `battery` being null is expected. 
The `battery` has never been reported since day one - if you search internet you would see issues like this since 2016, which is basically the day they launched the device.
The `action` OTOH would change to `bell1` or `bell2` when you press the doorbell button.
```
{
    "last_seen": "2023-02-03T04:52:33.811Z",
    "linkquality": 127,
    "action": null,
    "battery": null
}
```
